bindata/network/multus/multus.yaml

```diff
@@ -585,8 +585,30 @@ spec:
             SERVICEACCOUNT_TOKEN=$(cat $SERVICE_ACCOUNT_PATH/token)
             SKIP_TLS_VERIFY=${SKIP_TLS_VERIFY:-false}
 
+            LAST_SERVICEACCOUNT_MD5SUM=""
+            LAST_KUBE_CA_FILE_MD5SUM=""
 
-            # Check if we're running as a k8s pod.
+
+            # Setup our logging routines
+
+            function log()
+            {
+                echo "$(date --iso-8601=seconds) ${1}"
+            }
+
+            function error()
+            {
+                log "ERR:  {$1}"
+            }
+
+            function warn()
+            {
+                log "WARN: {$1}"
+            }
+
+
+            function generateKubeConfig {
+              # Check if we're running as a k8s pod.
             if [ -f "$SERVICE_ACCOUNT_PATH/token" ]; then
```

Comment:
hate to be this guy -- but, is this funky indentation expected?

Reply:
ah, might have missed a line when mass tabbing over